OpenRA.Mods.Common/AI/SupportPowerDecision.cs

```diff
@@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
 		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
 		public class Consideration
 		{
-			public enum DecisionMetric { Health, Value, None }
+			public enum DecisionMetric { Health, Value, BaseBuilding, None }
```

Comment:
Please do try and be careful with your wording @GraionDilach (you know what i'm talking about, even though you've since edited your comment).  Getting a first PR reviewed and merged is stressful enough for a new contributor, so the least we can do is to be polite.